docs/dev/plugins/cloudFoundry/deploy-cf-pcf-dev.md

```diff
@@ -0,0 +1,68 @@
+# Deploying CloudFoundry using Pivotal PCF Dev
+
+## Setup
+
+This guide assumes the following Prerequisites:
+- VT-x or AMD-v virtualization enabled
+- At least 3 GB of RAM and 500 GB of Space
```

Comment:
500 GB?!
![image](https://user-images.githubusercontent.com/2706976/33560791-089ddd30-d911-11e7-98b7-d128606882c7.png)

Maybe this is not enough but 500 GB is just plain overkill.